src/layouts/EditNav.jsx

```diff
@@ -96,21 +98,27 @@ export default class EditTree extends Component {
       tree, parentId: source.parentId, index: source.index,
     }).data.type;
 
-    const destinationParentType = this.getParentFromTreeposition({
+    const destinationParent = this.getParentFromTreeposition({
       tree, parentId: destination.parentId,
-    }).data.type;
-
-    // Rule 2)
-    if (!('index' in destination) && sourceParentType !== destinationParentType) return;
-
-    // Rule 3)
-    if (sourceItemType === 'collection' && destinationParentType !== 'section') return;
-    if (sourceItemType === 'thirdnav' && destinationParentType !== 'collection') return;
-
+    });
 
-    // console.log(tree.items[source.parentId].data, tree.items[destination.parentId].data);
-    // console.log(source, destination);
-    console.log(sourceItemType, destinationParentType);
+    const destinationParentType = destinationParent.data.type;
+
+    switch (sourceItemType) {
+      case 'page':
+      case 'collection-page':
+      case 'thirdnav-page':
+        // Rule 2)
+        if (!('index' in destination) && ['page', 'collection-page', 'thirdnav-page'].includes(destinationParentType)) return;
+        break;
```

Comment:
Remove unnecessary `break`

Reply:
linter throws a `no-fallthrough`

Reply:
ah my bad, i misread it since it's an `if` or `break`